src/inject/GPMInject/interface/mini.js

```diff
@@ -14,8 +19,15 @@ window.wait(() => {
 
   window.GPM.mini.on('enable', () => {
     Emitter.fire('mini', { state: true });
-    oldSize = remote.getCurrentWindow().getSize();
-    mainWindow.setSize(MINI_SIZE, MINI_SIZE);
+
+	// Save regular size/pos and restore mini size/pos.
+    regularSize = remote.getCurrentWindow().getSize();
+    regularPosition = remote.getCurrentWindow().getPosition();
+    mainWindow.setSize(...miniSize);
+    if (miniPosition) {
```

Comment:
You can remove this if check, there is no case where miniPosition won't be defined (assuming you add the two recoomended lines above